ui/spells/spells.scss

```diff
@@ -22,22 +22,24 @@
   margin-bottom: 6px;
   display: flex;
   align-items: center;
-}
 
-.spell-tabs>button[type="action"] {
-  border-bottom: 0px;
-}
+  >button[type="action"] {
+    border-bottom: 0px;
+  }
```

Comment:
I also started nesting classes. See end result in `main.css`
No difference from these changes.